app/controllers/groups/questions/written/answered_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Groups
+  module Questions
+    module Written
+      class AnsweredController < ApplicationController
+        before_action :data_check, :build_request
+
+        ROUTE_MAP = {
+          index:    proc { |params| Parliament::Utils::Helpers::ParliamentHelper.parliament_request.group_questions_written_answered.set_url_params({ group_id: params[:group_id]}) }
+        }.freeze
+
+        def index
+          @answering_body, @answers = Parliament::Utils::Helpers::FilterHelper.filter(@request, 'AnsweringBody', 'Answer')
+
+          @answering_body = @answering_body.first
+
+          @answers_grouped_by_date = Array(@answers).group_by { |answer| answer.answer_given_date }
```

Comment:
Is there a helper in parliament utils that does this functionality? What happens if `answer_given_date` is nil? I'm almost certain that in this case the item is removed from the returned array. Is this the functionality we expect? What to the designs show for items that don't have a date

Reply:
Also, does group_by also order them? What if they are in a random order - do the dates need to be in descending order?